packages/cbioportal-clinical-timeline/src/TimelineTrack.tsx

```diff
@@ -350,20 +357,23 @@ const TimelineItemWithTooltip: React.FunctionComponent<{
                     });
 
                     setTooltipUid(uid);
+
+                    store.setHoveredTooltipUid(uid);
+                    store.setMousePosition({
+                        x: e.pageX,
+                        y: e.pageY,
+                    });
                 }
-                store.setHoveredTooltipUid(uid);
-                store.setMousePosition({
-                    x: e.pageX,
-                    y: e.pageY,
-                });
             }}
             onMouseLeave={e => {
-                const uid = syncTooltipUid();
-
-                if (uid && !store.isTooltipPinned(uid)) {
-                    store.removeTooltip(uid);
-                    setTooltipUid(null);
-                }
+                setTimeout(() => {
```

Comment:
by using a timeout here, we can PIN the tooltip on entry into the tooltip. that way, when this executes, if the user is in the tooltip itself, the if statement at 371 will evaluate false (and not remove the tooltip)

Reply:
would be good to put this into a comment